EventFiltering/CMakeLists.txt

```diff
@@ -1,44 +1,48 @@
-# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
-# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
-# All rights not expressly granted are reserved.
+#Copyright 2019 - 2020 CERN and copyright holders of ALICE O2.
```

Comment:
I think that for some kind of reason your automatic formatting is messing up the formatting of the CMakeFile, can you try to fix this? (i.e. avoid modifying lines that are not directly connected with your code)